From e59c0a09a31fc6adc3bda0d1dc30c01d45fcabf6 Mon Sep 17 00:00:00 2001
From: Xiao Yang <yangx.jy@cn.fujitsu.com>
Date: Wed, 14 Jun 2017 15:39:09 +0800
Subject: [PATCH] syscalls/add_key02: add the "logon" key type

This key type is introduced in kernel:
'9f6ed2ca257f(keys: add a "logon" key type)'

Signed-off-by: Xiao Yang <yangx.jy@cn.fujitsu.com>
---
 testcases/kernel/syscalls/add_key/add_key02.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/testcases/kernel/syscalls/add_key/add_key02.c b/testcases/kernel/syscalls/add_key/add_key02.c
index 2e3308d..979f668 100644
--- a/testcases/kernel/syscalls/add_key/add_key02.c
+++ b/testcases/kernel/syscalls/add_key/add_key02.c
@@ -56,6 +56,7 @@ struct tcase {
 	{ "rxrpc",		64 },
 	{ "rxrpc_s",		 8 },
 	{ "user",		64 },
+	{ "logon",              64 },
 };
 #endif /* HAVE_LINUX_KEYCTL_H */
 
-- 
2.9.3

